source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRMediaTarget.mm

```diff
@@ -119,15 +119,13 @@ - (void)doSelectAction
                     [overlayview addSubview:poster];
 
                     // overlayview sit between AVPLayverViewController view's control view and content view, and here we add the poster.
-                    CGFloat heightToWidthRatio = 0.0f;
+                    CGFloat heightToWidthRatio = 0.75f;
```

Comment:
Before the change, if (image) and (image.size.width <=0) then, heightToWidthRatio=0;
After the change, it's 0.75

Reply:
0.75 is more reasonable default value as ratio.